server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyContestComparisonAudit.java

```diff
@@ -197,32 +204,38 @@
    * The number of two-vote understatements recorded so far.
    */
   @Column(nullable = false)
-  private Integer my_two_vote_under = 0;
+  private Integer my_two_vote_under_count = 0;
```

Comment:
Does this field name change also change the serialization?

Reply:
These objects are not serialized directly. You should not notice it. It does change database column names.

Reply:
Great.